djqs/.pre-commit-config.yaml

```diff
@@ -1,4 +1,4 @@
-exclude: '^docs/conf.py'
```

Comment:
Won't this ignore the contents of the djqs library? Is it because it's already covered by the top level pre-commit config?

Reply:
Hmm I think you're right. Let me fix this!